tests/dev/test-sets.R

```diff
@@ -1,10 +1,10 @@
 rm(list = ls())
 library(ospsuite)
-setwd("C:/Users/ahamadeh/Dropbox/rproject/workflow/")
+setwd("C:/Users/ahamadeh/Dropbox/GitHub/OSP/OSPSuite.ReportingEngine")
 devtools::load_all("C:/Users/ahamadeh/Dropbox/GitHub/OSP/OSPSuite.ReportingEngine")

```

Comment:
It would be great to ensure that there are no ABSOLUTE PATH in the examples
They should only use relative path 
By default the path is the package folder.